### ISSUE TITLE
Docs: Proposal to Add Vue SFC Code Block Linting Example

### ISSUE THREAD

**Issue by KumJungMin** [eslint/markdown#707]
### Docs page(s)

New example directory and files to be added:

- `examples/vue/README.md`
- `examples/vue/eslint.config.mjs`
- `examples/vue/package.json`

Reference existing examples:

- `examples/react`
- `examples/typescript`

### What documentation issue do you want to solve?

Currently, the `examples` directory includes React and TypeScript examples, 
but there is no example that demonstrates linting Vue SFC code blocks embedded in Markdown documents.

By combining the `@eslint/markdown` processor with `eslint-plugin-vue`, ESLint can lint `vue` code blocks like the following:

```vue
<template>
    <button @click="count++">
        {{ count }}
    </button>
</template>

<script setup>
const count = ref(0);
</script>
```

I would like to add an example that shows how Vue users can lint Vue example code written inside Markdown files, including the relevant configuration and usage instructions.

### What do you think is the correct solution?

I propose adding an `examples/vue` directory following the same structure as the existing React and TypeScript examples.

The example will include:

- A `README.md` containing Vue code blocks with intentional lint errors
- A flat config setup combining `@eslint/markdown` processor and `eslint-plugin-vue`
- A `package.json` to allow the example to be installed and run independently
- Instructions for running the example and expected ESLint output

If this direction is acceptable, I would like to proceed with the contribution :)

### Participation

- [x] I am willing to submit a pull request for this change.

### AI acknowledgment

- [ ] I did not use AI to generate this issue report.
- [x] (If the above is not checked) I have reviewed the AI-generated content before submitting. (_I used AI to translate my Korean draft into English_)

**Comment by DMartens** [comment]
Is there anything noteworthy about the proposed example for Vue which is not covered by the existing examples?
The React example shows how to set options for a code block ("**/*.md/*.js"),
while the TypeScript example shows how to target the code blocks when using a recommended config (by using `Array#map`).

**Comment by KumJungMin** [comment]
> Is there anything noteworthy about the proposed example for Vue which is not covered by the existing examples?
> The React example shows how to set options for a code block ("**/*.md/*.js"),
> while the TypeScript example shows how to target the code blocks when using a recommended config (by using `Array#map`).

-------

You're right. The proposed Vue example does not demonstrate any pattern unique to Vue.

I had misunderstood the `examples` directory as a collection of examples based on widely used frameworks. I now understand that its purpose is to demonstrate distinct configuration patterns rather than repeatedly adding similar framework-specific examples.

I'll keep that intention in mind and consider the purpose of the existing examples more carefully before making future proposals.

Thank you for the clarification! I’ve closed the issue. 🙂
